bin/promote-nightly-to-release

```diff
@@ -202,11 +202,13 @@ function statusline() {
 }
 
 statusline "Creating packaging branch..."
-git checkout -b "HHVM-${VERSION}"
+git checkout -B "HHVM-${VERSION}"
 statusline "Marking support for new DISTRO apt repo.."
 echo "DISTRO-${VERSION}" >> DEBIAN_REPOSITORIES
 git commit DEBIAN_REPOSITORIES -m "Adding DISTRO-${VERSION} apt repositories"
 statusline "Pushing packaging branch..."
+git rebase master
```

Comment:
Suppose `HHVM-4.200` was previously created and pushed, then a new commit x is pushed to `master`, then this script is executed again.

Now `git rebase master` would put the `Marking support for new DISTRO apt repo` commit onto commit X. Fortunately, `git pull --rebase origin "HHVM-${VERSION}"` would eliminate the duplicate commit and essentially put commit X on the original remote HHVM-4.200. As a result, the new HHVM-4.200 would look like the original HHVM-4.200 with a cherry-picked commit X.

Reply:
I just revered 3ec8c83692cc58d9eb691d0daa793e2e36d2d097, which was my local `master` commit. It was pushed to the `HHVM-4.155` branch because of the mechanism described above.

I revered it because it is slightly different from this PR and would create conflicts if we cherry-pick further commits to `HHVM-4.155`.

Reply:
Perhaps explicitly specify origin/master, not just master?